run-ci.sh

```diff
@@ -6,6 +6,7 @@ rm -rf test/out
 
 yarn install --frozen-lockfile
 yarn lint
+yarn test
```

Comment:
@rmberrios actually, instead of adding `yarn test`, just remove the conditional on line 11